integrationExamples/gpt/amp/remote.html

```diff
@@ -0,0 +1,207 @@
+<!doctype html>
+<html>
+<head>
+<meta charset="utf-8">
+<meta name="robots" content="noindex">
+<script>
+
+    /** AMP x-domain iframe custom source file
+     *  host this file on a remote domain from your AMP pages,
+     *  set the `amp-3p-iframe-src` meta tag on your AMP pages
+     *  (see `./amp_page.html`)
+     *
+     *
+     *  see "Enhance incoming ad configuration" section of AMP docs
+     *  https://www.ampproject.org/docs/reference/components/amp-ad
+     */
+
+    (function() {
+var v = location.search.substr(1);
+if (!(/^\d+(-canary)?$/.test(v))) return;
+var u = 'https://3p.ampproject.net/'+encodeURIComponent(v)+'/f.js';
+document.write('<script'+' src="'+encodeURI(u)+'"><'+'/script>');
+})();
+</script>
+<script>
+    // The Prebid global must match Prebid.js setting
+    var $$PREBID_GLOBAL$$ = pbjs;
+
+    // Standard Prebid.js setup
+    var rightSlotSizes = [[300, 250], [300, 600], [300, 250], [100, 100]];
+
+    var pbjs = pbjs || {};
+    pbjs.que = pbjs.que || [];
+    var PREBID_TIMEOUT = 3000;
+
+    var date = new Date().getTime();
+
+    // wrap the rest of the setup in a function that will be called by the
+    // AMP `draw3p` hook
+    function loadPrebidJS() {
+
+        setTimeout(function() {
+            return initAdserver;
+        }(), PREBID_TIMEOUT);
+
+
+        pbjs.que.push(function () {
+
+            pbjs.logging = true;
+
+            var adUnits = [
```

Comment:
can we clean this up a little bit so all publisher defined variables are declared in one spot and that's the only code that has to change on the page?